C920:
  brightness: -1
  contrast: -1
  saturation: -1
  sharpness: -1
  backlight_compensation: -1
  exposure_auto: -1
  exposure: -1
  focus_absolute: 0
  focus_auto: 0
  width: 1280
  height: 720
  fps: 15
  rotate: false
  camera_name: C920